akka-actor/src/main/scala/akka/actor/FSM.scala

```diff
@@ -322,6 +329,11 @@ trait FSM[S, D] extends Actor with Listeners with ActorLogging {
   val -> = FSM.->
 
   /**
+   * Unicode alias for ->, matches a (S, S) pair
+   */
+  val → = FSM.->
```

Comment:
actually we don't need this val – if instead the object is defined inside the class it all works fine and MiMa does not complain since there's no weird setter.

// Revived looking at this since bumping scalariform